src/middlewares/query-parser.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @description
+ * Create middleware for query parsing.
+ * a. Parsed query should be added to request stream object as parsedQuery field.
+ *
+ * @param req
+ * @param res
+ * @param next
+ */
+
+
+/**
+ * @description
+ * Middleware for query parsing in request.
+ * Parse query string and populate `req.parsedQuery`
+ * with an object keyed and value by the query data.
+ *
+ * @param {Request} req
+ * @param {http.ServerResponse} res
+ * @param {Function} next
+ */
+export const queryParser = (req, res, next) => {
```

Comment:
This solution is difficult. Please use https://nodejs.org/docs/latest/api/url.html#url_url_parse_urlstring_parsequerystring_slashesdenotehost with true flag

Reply:
Done)) I thought that we need to write it by ourselves.